<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="message_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>651</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="main_user_input">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="user_id">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>user_id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>520</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="connected_users_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>91</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="host_list_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionconnect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionconnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
